src/pages/home/HomePage.js

```diff
@@ -254,6 +275,8 @@ class App extends React.Component {
                                         insets={insets}
                                         onLinkClick={this.recordTimerAndToggleHamburger}
                                         isChatSwitcherActive={this.props.isChatSwitcherActive}
+                                        fabAnimatedValue={this.animationFabActivate}
+                                        fabOnPress={this.toggleFab}
```

Comment:
Maybe more consistent with our naming to call this `onFloatingActionButtonPress`

Reply:
Changed it. 
Because I am not passing the Animated value anymore, I called the other prop `isFloatingActionButtonActive`.